AI/VCAI/Goals.cpp

```diff
@@ -883,43 +879,43 @@ TSubgoal GatherTroops::whatToDoToAchieve()
 				continue;
 
 			BuildingID bid(BuildingID::DWELL_FIRST + creature->level - 1 + upgradeNumber * GameConstants::CREATURES_PER_TOWN);
-			if (t->hasBuilt(bid)) //this assumes only creatures with dwellings are assigned to faction
+			if(t->hasBuilt(bid)) //this assumes only creatures with dwellings are assigned to faction
 			{

```

Comment:
this change only fixed spacebar - excessive { } is separate problem. This commit only changed whitespaces.